sys/net/network_layer/sixlowpan/ip.c

```diff
@@ -44,12 +44,12 @@ char addr_str[IPV6_MAX_ADDR_STR_LEN];
 #define LLHDR_IPV6HDR_LEN           (LL_HDR_LEN + IPV6_HDR_LEN)
 #define IPV6_NET_IF_ADDR_BUFFER_LEN (NET_IF_MAX * IPV6_NET_IF_ADDR_LIST_LEN)
 
-uint8_t ip_send_buffer[BUFFER_SIZE];
+static uint8_t ip_send_buffer[BUFFER_SIZE];
 uint8_t buffer[BUFFER_SIZE];
```

Comment:
This buffer is not very well named, maybe `sixlowpan_buffer` is a more fitting name?
OT: This is one of the oldest lines in this file.

https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#naming